src/app/base/components/record-card/record-card.component.ts

```diff
@@ -0,0 +1,70 @@
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+import { DataBrowserModals, RecordCommentsModal } from 'src/app/data/components/data-browser/helper-classes/modals-helper';
+
+
+@Component({
+  selector: 'kern-record-card',
+  templateUrl: './record-card.component.html',
+  styleUrls: ['./record-card.component.scss']
+})
+export class RecordCardComponent implements OnInit {
+
+  @Input() record: Record;
+  @Input() attributesSortOrder: AttributeSort;
+  @Input() index: number;
+  @Input() extendedRecords: ExtendedRecord;
+  @Input() similarSearchHelper: any;
+  @Input() recordComments: RecordCommentsModal;
+  @Input() attributes: Map<string, any>;
+  @Input() dataBrowserModals: DataBrowserModals;
+  @Input() activeSearchParams: any;
+
+  @Output() preliminaryRecordIds = new EventEmitter<number>();
+
+  constructor() { }
+
+  ngOnInit(): void {
+  }
+
+  storePreliminaryRecordIds(index: number) {
+    this.preliminaryRecordIds.emit(index);
```

Comment:
why is that emitted? if this only stores it in the local storage so maybe no need to emit?

Reply:
That's true, no need for emitting, moved the logic from the parent component